### ISSUE TITLE
omacro doesn't work in .csound6rc

### ISSUE THREAD

**Issue by julianbpeterson** [csound/csound#1705]
Defining an omacro
--omacro:FOO=BAR
in csound6rc does not seem to work.

instr 1
	printf_i "$FOO \n", 1
endin

will result in 
BAR 
if the omacro is specified on the command line or in <CsOptions>, but when defined in the resource file the result is
$error

**Comment by jpffitch** [comment]
Yes; this is by design.   I have tried allowing macro defnitions in .csoung?rc and it seems to work.  I canot remember why that was not allowed.  I will seek advice
